PPI/manual_work/May_2023/PPI_concepts_correction_May2023.sql

```diff
@@ -0,0 +1,23 @@
+-- Update two concepts and set for 2 concepts diffirent concept_class_id
```

Comment:
Typo here. The correct one would sound like this:
Update two concepts and set different concept_class_id for 2 concepts